docs/api.rst

```diff
@@ -199,6 +199,11 @@ and
 
 define the same simulation.
 
+.. note:: When ``Ne`` and an instance of :class:`.SimulationModel` with
+    ``reference_size`` set are both provided as arguments to :func:`.simulate`,
+    the ``Ne`` parameter is ignored.
```

Comment:
shouldn't this throw a warning at least?

Reply:
Can't I'm afraid, we have no way of detecting it because the default for Ne is 1 in the top-level function. Changing this to None could have all sorts of subtle side effects, which I don't want to get into, tbh.